data/korath/korath ships.txt

```diff
@@ -15,58 +15,57 @@ ship "Korath Dredger"
 	attributes
 		category "Heavy Freighter"
 		"cost" 29370000
-		"shields" 39000
-		"hull" 16000
-		"required crew" 201
-		"bunks" 395
-		"mass" 1503
-		"drag" 18.3
-		"heat dissipation" .65
+		"shields" 27000
+		"hull" 22000
+		"required crew" 137
+		"bunks" 331
+		"mass" 1357
+		"drag" 17.1
+		"heat dissipation" .45
 		"fuel capacity" 900
 		"cargo space" 605
-		"outfit space" 827
-		"weapon capacity" 297
-		"engine capacity" 181
-		
+		"outfit space" 607
```

Comment:
```suggestion
		"outfit space" 757
```

I know I suggested dropping the outfit and weapon space a bit, but I admit I wasn't envisaging a 220 drop. The nerf to the weapon capacity looks good, but I think the outfit space should be moved back up, at least to 707, preferably closer to 750. Ideally, this would have a range of the "civilian" outfits taking up a bunch of that space. Enough to support some degree of self-sufficiency, at least over the short to medium term. 

Even if they are, strictly speaking, capable of being reduced all the way down to 600; I would prefer that they remain an interesting and viable general purpose ship that the player can use for a variety of purposes. I don't want them to become just a trophy or short-term step on a capture ladder. At the end of the day, that involves allowing them enough space to have some flexibility.